test/e2e/man.cjs

```diff
@@ -42,6 +42,7 @@ The animation of the selected test case will be displayed using the chosen Vizzu
 			'/test/e2e/tests/style_tests.json',
 			'/test/e2e/tests/features.json',
 			'/test/e2e/tests/fixes.json',
+			'/test/e2e/tests/tickets.json',
```

Comment:
lehet szerencsésebb lenne a bugs.json elnevezés. a fixes.json-ben is ticketek szerint vannak. illetve a features/fixes/... listába is jobban illik szerintem.

Reply:
Csak az a baj hogy ezek nem fix-ek, ezért szerettem volna külön tenni (és jó is külön kezelni őket). Viszont bugoknak meg nem akartam nevezni, mert az azt is jelentené mintha ezek lennének "csak" a bugjaink, és nem is mindenhol egyértelmű hogy mi a hiba/hiba van. Nem találtam neki megfelelő nevet, ezért maradtam a tickets-nél mint általánosan valami pointerek. Lehetnének még pl faults-ok, de találhatunk rá jobb nevét is

Reply:
ok, szerintem mergelhető így